docs/build-instructions/linux.md

```diff
@@ -128,13 +132,15 @@ See also [#2082](https://github.com/atom/atom/issues/2082).
 ### /usr/bin/env: node: No such file or directory
 
 If you get this notice when attempting to `script/build`, you either do not
-have Node.js installed, or node isn't identified as Node.js on your machine.
-If it's the latter, entering `sudo ln -s /usr/bin/nodejs /usr/bin/node` into
+have Node.js installed, or node is not identified as Node.js on your machine.
+If it is the latter, entering `sudo ln -s /usr/bin/nodejs /usr/bin/node` into
```

Comment:
Agreed. @fusion809 In general, it makes a PR easier to merge if you make one focused change. So in this case, could you undo the minor wording changes that you made to the surrounding prose?